jetty-util/src/main/java/org/eclipse/jetty/util/IO.java

```diff
@@ -474,7 +474,7 @@ public static void close(Writer writer)
      * For Windows / Dos, that means {@link java.nio.file.attribute.DosFileAttributes}
      * </p>
      */
-    public static FileAttribute<?>[] getUserOnlyFileAttribute(Path path)
+    public static FileAttribute<?>[] getUserPrivateFileAttribute(Path path)
```

Comment:
I need to test a few things on OSX and Windows, but I think I can get rid of this entire method.